share/lib/python/scripts/_binwrapper.py

```diff
@@ -91,6 +83,15 @@ def _config_exe(exe_name):
     os.environ["CORENRN_PERLEXE"] = shutil.which("perl")
     os.environ["NRNBIN"] = os.path.dirname(__file__)
 
+    if "NMODLHOME" not in os.environ:
+        os.environ["NMODLHOME"] = NRN_PREFIX
+    if "NMODL_PYLIB" not in os.environ:
+        os.environ["NMODL_PYLIB"] = find_libpython()
+
+    # nmodl module is inside <prefix>/lib directory
+    sys.path.insert(0, os.path.join(NRN_PREFIX, "lib"))
+    os.environ["PYTHONPATH"] = ":".join(sys.path)
```

Comment:
does this work in Windows? Usually in path entries are separated by semicolons, but I don't know if Python standardizes that.

Reply:
Good catch. I'm trying
```
os.environ["PYTHONPATH"] = os.pathsep.join(sys.path)
```
But it is a puzzle why it did not get caught with ```nrniv -python -c 'import neuron; neuron.test(); quit()'``` when python is a virtual environment. I'll look into it further on Monday.

Reply:
Windows is not using this file (or any of the copies installed in c:/nrn/lib/python/scripts). Those are only used for wheels.